scripts/irods/test/test_control_plane.py

```diff
@@ -82,6 +84,92 @@ def test_shutdown_local_server(self):
             finally:
                 IrodsController().start()
 
+    def test_shutdown_local_server_using_hostname_alias_defined_hosts_config__issue_5700(self):
+        try:
+            with lib.file_backed_up(paths.hosts_config_path()):
+                hostname_alias = 'hostname-alias.issue-5700'
+
+                with open(paths.hosts_config_path(), 'r+') as f:
+                    hosts_config = json.load(f)
+                    hosts_config['host_entries'] = [{
+                        'address_type': 'local',
+                        'addresses': [
+                            {'address': lib.get_hostname()},
+                            {'address': hostname_alias}
+                        ]
+                    }]
+
+                    f.seek(0)
+                    f.truncate(0)
+                    f.write(json.dumps(hosts_config, indent=4))
+
+                # Restart the server so that the origin server process sees the updates to
+                # the hosts_config.json file.
+                IrodsController().restart()
+
+                with session.make_session_for_existing_admin() as admin_session:
+                    admin_session.environment_file_contents = IrodsConfig().client_environment
+                    initial_size_of_server_log = lib.get_file_size_by_path(IrodsConfig().server_parent_log_path)
+
+                    expected_output = [json.dumps({'hosts': [{'shutting down': lib.get_hostname()}]}, indent=4)]
+                    admin_session.assert_icommand(['irods-grid', 'shutdown', '--hosts', hostname_alias], 'STDOUT', expected_output)
+                    lib.delayAssert(
+                        lambda: lib.log_message_occurrences_equals_count(
+                            msg='iRODS Server is done',
+                            server_log_path=IrodsConfig().server_parent_log_path,
+                            start_index=initial_size_of_server_log))
+
+        finally:
+            # Restart the server so that the origin server process sees the original
```

Comment:
here, you've already got origin AND original.    consider a rewording...

Reply:
but *original* refers to the hosts_config.json file.

Reply:
right.    so, can you change "origin server" to ... "initial server"   and/or "initial server (parent of the agent factory)"?

Reply:
done.